pkg/cmd/cli/cmd/newapp.go

```diff
@@ -423,6 +424,10 @@ func setupAppConfig(f *clientcmd.Factory, out io.Writer, c *cobra.Command, args
 	// Only output="" should print descriptions of intermediate steps. Everything
 	// else should print only some specific output (json, yaml, go-template, ...)
 	output := cmdutil.GetFlagString(c, "output")
+
+	if output == "wide" {
+		return fmt.Errorf("output \"wide\" is undefined")
```

Comment:
there is, but it's tricky i guess and technically `wide` doesn't even do anything?
check this out: https://github.com/kubernetes/kubernetes/blob/00c754108fc1d8d6b0e353b8ad6781ba75ac24d8/pkg/kubectl/resource_printer.go#L125